cloudformation/database/serverless-database.yaml

```diff
@@ -0,0 +1,345 @@
+        - Key: 'parallel-cluster:accounting:version'
+          Value: '1.0'
+      VpcId: !If [CreateVPC, !Ref ServerlessDatabaseClusterVpc, !Ref Vpc]
+  ServerlessDatabaseClusterSecurityGroupInboundRule:
+    Type: 'AWS::EC2::SecurityGroupIngress'
+    Properties:
+      IpProtocol: tcp
+      Description: Allow incoming connections from client security group
+      FromPort: !GetAtt
+        - ServerlessDatabaseCluster
+        - Endpoint.Port
+      GroupId: !GetAtt
+        - ServerlessDatabaseClusterSecurityGroup
+        - GroupId
+      SourceSecurityGroupId: !GetAtt
+        - ServerlessClusterDatabaseClientSecurityGroup
+        - GroupId
+      ToPort: !GetAtt
+        - ServerlessDatabaseCluster
+        - Endpoint.Port
+  ServerlessDatabaseCluster:
+    Type: 'AWS::RDS::DBCluster'
+    Properties:
+      Engine: aurora-mysql
+      CopyTagsToSnapshot: true
+      DBClusterParameterGroupName: !Ref ServerlessDatabaseClusterParameterGroup
+      DBSubnetGroupName: !Ref ServerlessDatabaseClusterSubnetGroup
+      EnableHttpEndpoint: false
+      EngineMode: serverless
+      MasterUsername: !Ref ClusterAdmin
+      MasterUserPassword: !Sub '{{resolve:secretsmanager:${AdminPasswordSecret}:SecretString:::}}'
+      ScalingConfiguration:
+        AutoPause: true
+        MaxCapacity: !Ref MaxCapacity
+        MinCapacity: !Ref MinCapacity
+        SecondsUntilAutoPause: 600
+      StorageEncrypted: true
+      Tags:
+        - Key: 'parallel-cluster:accounting'
+          Value: database
+        - Key: 'parallel-cluster:accounting:scheduler'
+          Value: slurm
+        - Key: 'parallel-cluster:accounting:version'
+          Value: '1.0'
+      VpcSecurityGroupIds:
+        - !GetAtt
+          - ServerlessDatabaseClusterSecurityGroup
+          - GroupId
+    UpdateReplacePolicy: Snapshot
+    DeletionPolicy: Snapshot
+  ServerlessClusterDatabaseClientSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Security Group to allow connection to Serverless DB Cluster
+      Tags:
+        - Key: 'parallel-cluster:accounting'
+          Value: client-security-group
+        - Key: 'parallel-cluster:accounting:scheduler'
+          Value: slurm
+        - Key: 'parallel-cluster:accounting:version'
+          Value: '1.0'
+      VpcId: !If [CreateVPC, !Ref ServerlessDatabaseClusterVpc, !Ref Vpc]
+  ServerlessClusterDatabaseClientSecurityGroupOutboundRule:
+    Type: 'AWS::EC2::SecurityGroupEgress'
+    Properties:
+      GroupId: !GetAtt
+        - ServerlessClusterDatabaseClientSecurityGroup
+        - GroupId
+      IpProtocol: tcp
+      Description: Allow incoming connections from PCluster
+      DestinationSecurityGroupId: !GetAtt
+        - ServerlessDatabaseClusterSecurityGroup
+        - GroupId
+      FromPort: !GetAtt
+        - ServerlessDatabaseCluster
+        - Endpoint.Port
+      ToPort: !GetAtt
+        - ServerlessDatabaseCluster
+        - Endpoint.Port
+Outputs:
```

Comment:
General comment on `Outputs`:
1. why should we declare `Exports: Name` instead of simply setting their name with the key?
I mean having this:

```
Outputs:
  ParameterName:
    Value: ParameterValue
```

instead of:

```
Outputs:
  OutputName:
    Value: OutputValue
    Exports:
      Name: AnotherOutputName
```

1. I think it's not necessary to prefix all outputs with `parallelcluster` because (i) the stack does not contain anything specific to pcluster and (ii) even if the stack would be really specific to pcluster, we can consider it implicit.